### PR TITLE
Spot Bugs / Sonarqube 

### PR DESCRIPTION
**Pull request by tnfru**

### CODE REVIEW

File: src/main/java/de/hhu/propra/sharingplatform/faker/DataFaker.java [hunk 1]
```diff
@@ -45,11 +44,10 @@
     private Faker faker;
 
     @Autowired
-    public DataFaker(Environment env, UserRepo userRepo,
-        ItemRepo itemRepo, OfferRepo offerRepo,
-        OfferService offerService,
-        IPaymentApi apiService) {
-        this.env = env;
```
Comment (tnfru) [inline src/main/java/de/hhu/propra/sharingplatform/faker/DataFaker.java:52]: is this safe to remove @brokelyn ?

Comment (brokelyn) [reply src/main/java/de/hhu/propra/sharingplatform/faker/DataFaker.java:52]: currently yes.

File: src/test/java/de/hhu/propra/sharingplatform/controller/OfferControllerTest.java [hunk 2]
```diff
@@ -276,10 +278,14 @@ public void offerRequestPostLoggedInValid() throws Exception {
...
             .create(a1.capture(), a2.capture(), a3.capture(), a4.capture());
 
-        assertEquals(a1.getValue().longValue(), 1337);
-        assertEquals(a2.getValue(), user);
-        assertEquals(a3.getValue(), LocalDateTime.of(2019, 2, 23, 0, 0, 0));
-        assertEquals(a4.getValue(), LocalDateTime.of(2019, 2, 27, 23, 59, 59));
+        assertEquals(1337, a1.getValue().longValue());
+        assertEquals(user, a2.getValue());
+        assertEquals(LocalDateTime
+                .of(2019, 2, 23, 0, 0, 0),
```
Comment (brokelyn) [inline src/test/java/de/hhu/propra/sharingplatform/controller/OfferControllerTest.java:284]: Why line break?

**Review by brokelyn — APPROVED** [review]
Verified!

**Pull request merged**